requirements.txt

```diff
@@ -3,3 +3,7 @@ Flask-Script==2.0.5
 
 # Elasticsearch 1.0
 elasticsearch>=1.0.0,<2.0.0
+
+#Required for testing
```

Comment:
Good point. I was following the convention in my head rather than that documented in the README. My fail but having it the same across apps would mean that kind of mistake wouldn't happen again.